output_basic.go

```diff
@@ -79,32 +79,32 @@ func BasicOutputUnlockConditionsArrayRules() serializer.ArrayRules {
 	return *basicOutputUnlockCondsArrayRules
 }
 
-// BasicOutputFeatureBlocksArrayRules returns array rules defining the constraints on FeatureBlocks within an BasicOutput.
-func BasicOutputFeatureBlocksArrayRules() serializer.ArrayRules {
+// BasicOutputFeaturesArrayRules returns array rules defining the constraints on Features within an BasicOutput.
+func BasicOutputFeaturesArrayRules() serializer.ArrayRules {
 	return *basicOutputFeatBlockArrayRules
 }
 
 // BasicOutputs is a slice of BasicOutput(s).
 type BasicOutputs []*BasicOutput
 
-// BasicOutput is an output type which can hold native tokens and feature blocks.
+// BasicOutput is an output type which can hold native tokens and features.
 type BasicOutput struct {
 	// The amount of IOTA tokens held by the output.
 	Amount uint64
 	// The native tokens held by the output.
 	NativeTokens NativeTokens
 	// The unlock conditions on this output.
 	Conditions UnlockConditions
-	// The feature blocks which extending the output metadata.
-	Blocks FeatureBlocks
+	// The features on the output.
+	Feats Features
```

Comment:
Call it `Features` because a feat is something else

Reply:
The reason I've done this is because of the naming clash with the interface `Output`'s `Features()` function. However, I changed these interface functions to return sets (as one would want to get the set anyways), therefore adjusted the names and reverted back the fields within the output structs to `Features` etc.